docs/src/main/asciidoc/spring-cloud-commons.adoc

```diff
@@ -251,6 +251,9 @@ This behavior can be disabled by setting `autoRegister=false` in `@EnableDiscove
 NOTE: `@EnableDiscoveryClient` is no longer required.
 You can put a `DiscoveryClient` implementation on the classpath to cause the Spring Boot application to register with the service discovery server.
 
+NOTE: Since 2.2.0 we also provide a `ReactiveDiscoveryClient` interface.
```

Comment:
We tend not to mention versions in the doc. Maybe something like. "A `ReactiveDiscoveryClient` interface is now provided as well."